src/library/scala/collection/convert/Wrappers.scala

```diff
@@ -411,24 +411,21 @@ private[collection] trait Wrappers {
 
     override def size = underlying.size
 
-    def get(k: String) = {
-      val v = underlying get k
-      if (v != null) Some(v.asInstanceOf[String]) else None
+    def get(k: String): Option[String] = {
+      Option((underlying get k).asInstanceOf[String])
```

Comment:
In [Option.scala](https://github.com/scala/scala/blob/2.11.x/src/library/scala/Option.scala#L25) the same code:
```scala
if (x == null) None else Some(x)
```
I don't see the reason why is should be slower.

Of course, there are function calling... Is that so expensive?